guides/hack/10-types/79-type-aliases.md

```diff
@@ -33,8 +38,9 @@ function distance(Point $p1, Point $p2): float {
 }
 ```
 
-Being in the same source file as the alias definition, the functions `create_Point` and `distance` have direct access to the integer
-fields in any `Point`'s tuple.  However, any file that includes this file does not.
+## `Type` v. `Newtype` | (Trasparent v. Opaque)
```

Comment:
Thank you for fixing all these issues. This typo jumped out at me.

```suggestion
## `Type` v. `Newtype` | (Transparent v. Opaque)
```

Reply:
Thanks! Law of Conservation of Typos: they can not be created or destroyed...